app/views/shares/show.html.erb

```diff
@@ -1,12 +1,19 @@
 <div class="booyah-box col-xs-10 col-xs-offset-1">
   <h1><%= @share.title %></h1><br />
-  <i><%= current_user.name %></i><br /><br />
+  <i><%= @share.user.name %></i><br /><br />
 
   <p><%= linkify_hashtags(@share.body) %></p><br />
   <%= pluralize(@share.votes.count, "votes") %>
   <%= button_to '+1', upvote_share_path(@share), method: :post %><br />
-  <div class="btn btn-warning pull-right"><%= link_to 'Edit', edit_share_path(@share) %></div>
-  <div class="btn btn-warning pull-right"><%= link_to 'Delete', share_path(@share), method: :delete, data: {confirm: 'Are you sure?'} %></div>
+
+  <% if current_user == @share.user %>

```

Comment:
Since this is used in multiple places, might make sense to build a helper method in `application_helper.rb`  Something like:

```
def current_user_created_share?(share)
  current_user == share.user
end
```

This is pretty minor though, but might be helpful since it's in a few places.